themes-default/slim/views/partials/showheader.mako

```diff
@@ -94,45 +77,46 @@
-                                     <img alt="[imdb]" height="16" width="16" src="images/imdb.png" style="margin-top: -1px; vertical-align:middle;"/>
-                                 </app-link>
-                 % endif
-                % if show.externals.get('trakt_id'):
-                    <app-link href="https://trakt.tv/shows/${show.externals.get('trakt_id')}" title="https://trakt.tv/shows/${show.externals.get('trakt_id')}">
+                    <span v-if="show.rating.imdb && show.rating.imdb.rating"
+                        class="imdbstars"
+                        :qtip-content="show.rating.imdb.rating + ' / 10 Stars<br> ' + show.rating.imdb.votes + ' Votes'"
+                    >
+                        <span :style="{ width: (Number(show.rating.imdb.rating) * 12) + '%' }"></span>
+                    </span>
+                    <template v-if="!show.rating.imdb">
+                        <span v-if="show.year && show.year.start">({{ show.year.start }}) - {{ show.runtime }} minutes - </span>
+                    </template>
+                    <template v-else>
+                        <template v-if="show.country_codes && show.country_codes.length >= 1 ">
+                            <img v-for="country in show.country_codes" src="images/blank.png" :class="['country-flag', 'flag-' + country]" width="16" height="11" style="margin-left: 3px; vertical-align:middle;" />
+                        </template>
+                        <span>
+                        % if show.imdb_info.get('year'):
+                            (${show.imdb_info['year']}) -
+                        % endif
+                            ${show.imdb_info.get('runtimes') or show.runtime} minutes
+                        </span>
+                        <app-link :href="'http://www.imdb.com/title/' + show.id.imdb" :title="'http://www.imdb.com/title/' + show.id.imdb">
+                            <img alt="[imdb]" height="16" width="16" src="images/imdb.png" style="margin-top: -1px; vertical-align:middle;"/>
+                        </app-link>
+                    </template>
+                    <app-link v-if="show.id.trakt" :href="'https://trakt.tv/shows/' + show.id.trakt" :title="'https://trakt.tv/shows/' + show.id.trakt">
                         <img alt="[trakt]" height="16" width="16" src="images/trakt.png" />
                     </app-link>
-                % endif
-                 <app-link href="${indexerApi(show.indexer).config['show_url']}${show.indexerid}" title="${indexerApi(show.indexer).config["show_url"] + str(show.indexerid)}">
-                     <img alt="${indexerApi(show.indexer).name | h}" height="16" width="16" src="images/${indexerApi(show.indexer).config["icon"]}" style="margin-top: -1px; vertical-align:middle;"/>
-                 </app-link>
+                    ## @TODO: What is this?
```

Comment:
The link to the show on the indexer website.
![image](https://user-images.githubusercontent.com/10238474/44861541-66c8e280-ac81-11e8-9891-cb915d1cba23.png)